Source/MediaInfo/Audio/File_Usac.h

```diff
@@ -121,8 +128,29 @@ public :
     //Elements - USAC - Frame
     #if MEDIAINFO_TRACE || MEDIAINFO_CONFORMANCE
     void UsacFrame                          (size_t BitsNotIncluded=(size_t)-1);
-    void UsacSingleChannelElement           ();
-    void UsacChannelPairElement             ();
+
+    void twData                             ();
+    void icsInfo                            ();
+    void scaleFactorData                    ();
+    void arithData                          (size_t ch, int16u N, int16u lg, bool arith_reset_flag);
+    void acSpectralData                     (size_t ch, bool usacIndependencyFlag);
+    void fdChannelStream                    (size_t ch, bool commonWindow, bool commonTw, bool tnsDataPresent, bool usacIndependencyFlag);
+    void cplxPredData                       (int8u max_sfb_ste, bool usacIndependencyFlag);
+    void StereoCoreToolInfo                 (bool& tns_data_present0, bool& tns_data_present1, bool core_mode0, bool core_mode1, bool usacIndependencyFlag);
+    void UsacCoreCoderData                  (size_t nrChannels, bool usacIndependencyFlag);
+    void UsacSingleChannelElement           (bool usacIndependencyFlag);
+    void UsacChannelPairElement             (bool usacIndependencyFlag);

```

Comment:
Let the 2 functions already there on the top for better diff (same for .cpp), no empty line.